scrapers/wer.py

```diff
@@ -0,0 +1,88 @@
+import os
+import re
+import string
+import urllib2
+
+from BeautifulSoup import BeautifulSoup
+from dateutil import parser
+import mistune
+
+from data.acoustics import speech_recognition
+from scales import error_rate # or error_percent?
+from taxonomy import offline
+
+# needs to be manually updated when more metrics are added
+h2_name_map = {
+        "LibriSpeech" : "librispeech",
+        "WSJ" : "wsj",
+        "Switchboard Hub5'00" : "swb_hub_500",
+        "Fisher" : "fisher",
+        "CHiME (noisy speech)" : "chime",
+        "TIMIT" : "timit"
+        }
+
+def get_metrics(h2_name, first_row, file_output, scale = "error_percent"):
+    metric_names = []
+    name = h2_name_map[h2_name]
+    if name == "":
+        raise SystemExit("Error! Need to add h2_name to h2_name_map to be able to parse!")
+    for column in first_row:
+        column = str(column.getText()).translate(None, string.punctuation)
+        metric_name = str(name + "_" + "_".join(re.findall(r"[\w']+", column))).replace("test", "")
+        metric_names.append(metric_name)
+        s = "{0} = speech_recognition.metric(name=\"{1} {2}\", scale={3}, attributes=['language'])\n".format(metric_name, name, column, scale)
+        file_output += s
+    return (metric_names, file_output)
+
+def add_measures(metric_names, row):
+    def row_data(row):
+        columns = row.findAll('td')
+        l = len(columns)
+        url = columns[l-3].find('a', href=True)['href']
+        columns = map(lambda x: x.getText(), columns)
+        date = str(parser.parse(columns[l-2]).date()).split("-")
+        date = "date({0}, {1}, {2})".format(int(date[0]), int(date[1]), int(date[2]))
+        r = {
+                'name': columns[l-1].encode('ascii', 'ignore'),
+                'date': date,
+                'url': url,
+                'values': map(lambda x: x.strip('%'), columns[:-3])
+            }
+        return r
+    data = row_data(row)
+    notes = data['name']
+    if notes == "Humans":
+        # TODO: eventually use the human measurements as benchmarks. That's kind of hard right now.
```

Comment:
Yes, I know that it's easy to add a `target` parameter to `.metric` call, but this comment was referencing a different issue. Implementing this right requires a bit of restructuring, because by the time those targets are read in the lower half of the table, the metrics have already been written. I'll fix this soon.